webpack.config.js

```diff
@@ -146,6 +150,24 @@ const webpackConfigLoaders = [
 const webpackConfigPlugins = [
 
   /**
+   * Plugin: DefinePlugin
+   * Description: Define free variables.
+   * Useful for having development builds with debug logging or adding global constants.
+   *
+   * Environment helpers
+   *
+   * See: https://webpack.github.io/docs/list-of-plugins.html#defineplugin
+   */
+  // NOTE: when adding more properties, make sure you include them in globals.d.ts
+  new DefinePlugin( {
+    ENV: JSON.stringify( METADATA.ENV ),
```

Comment:
without `JSON.stringify` it doesn't work and app is broken.
They are stringifying it also in docs
https://webpack.github.io/docs/list-of-plugins.html#defineplugin
very strange :-|